### PR TITLE
cg87.c: more conversion to CodeBuilder

### PR DESCRIPTION
**Pull request by WalterBright**
This is a refactor.

### CODE REVIEW

**Comment by WalterBright** [comment]
ping @andralex @MartinNowak

File: src/backend/cg87.c [hunk 1]
```diff
@@ -1225,52 +1217,61 @@ code *orth87(elem *e,regm_t *pretregs)
...
-            c2 = loadComplex(e2);
-            c3 = makesure87(e1, sz2, 2, 0);
-            c3 = cat(c3,makesure87(e1, 0, 3, 0));
-            retregs = mST01;
+        {
+            CodeBuilder cdb;
+            cdb.append(loadComplex(e1));
+            cdb.append(loadComplex(e2));
+            cdb.append(makesure87(e1, sz2, 2, 0));
+            cdb.append(makesure87(e1, 0, 3, 0));
```
Comment (andralex) [inline src/backend/cg87.c]: From the cycle "I have a dream": `append` would take variadic arguments.

**Review by andralex — APPROVED** [review]
alrighty it does improve matters

**Comment by andralex** [comment]
Auto-merge toggled on

**Pull request merged**